apps/web/public/static/locales/fr/common.json

```diff
@@ -2218,15 +2218,15 @@
   "not_enough_available_seats_error": "La réservation n'a pas assez de places disponibles",
   "user_redirect_title": "{{username}} est actuellement en absence de courte durée.",
   "user_redirect_description": "Pendant ce temps, {{profile.username}} sera responsable de tous les rendez-vous programmés de la part de {{username}}.",
-  "out_of_office": "Pas au bureau",
+  "out_of_office": "En dehors du bureau",
```

Comment:
I feel like in french "absent" for out of office it's the most common way to say it

Reply:
Thank for your feedback @ThyMinimalDev 

As a French speaker, "absent" is commonly used similarly to "unavailable," but "out of the office" directly translates to "En dehors du bureau," specifying absence from the workplace.

Reply:
Hey there, not sure about this, I am a french speaker myself too and I have never heard or seen this expression, being "out-of-office / OOO" does not directly translate to being "outside of the office" it means you are unavailable for meetings / work in general. so in the specific context of a calendar / booking system this being translated as "Absent du bureau" makes more sense.

can you make the change please and I'll approve your pr 👍 

Thanks

Reply:
Hi, I'm a French speaker as well and would have translated this simply to "Absences" considering where it is being used in the UI. Unlike OOO in English, "Absent du bureau" isn't a very common thing to read or hear so I think "Absences" better indicates what that part of the settings does. Cheers